src/bp/ui-studio/src/web/components/Layout/index.jsx

```diff
@@ -25,10 +27,13 @@ import { viewModeChanged } from '~/actions'
 import layout from './Layout.styl'
 import style from './style.scss'
 import StatusBar from './StatusBar'
+import EventEmitter2 from 'eventemitter2'
 
 class Layout extends React.Component {
+  statusBarEmitter = new EventEmitter2({ wildcard: true })
```

Comment:
do we really need an eventEmitter only to react after some event ? I mean we could simply pass down a `fireModuleEvent` func to modules components and push newEvents to  the status bar

Reply:
<discussed offline>